### PR TITLE
update document to verify from software public on repositories to verifier sc deployed

### PR DESCRIPTION
**Pull request by zkronos73**

### CODE REVIEW

**Review by eduadiez — DISMISSED** [review]

File: verifyMainnetDeployment/verifyMainnetProofVerifier.md [hunk 1]
```diff
@@ -213,18 +213,18 @@ To download bytecode of deployed smartcontract, need the address of smart contra
 
 Go to Etherscan or Beaconcha to get transaction bytecode.
 
-Associated with address _0x0775e11309d75aA6b0967917fB0213C5673eDf81_ found the transacction _0x99c654b2338dc6e9f438b82acd5eb8af2fa2d2fe69a387714f2b2fa935ee8dbe_.
+Associated with address _0x082cCe3072A26a3871D3e5D40afB425fF5038Cf6_ found the transacction _0xeef8a49cc2469c11043eeb4a1a90c9c184ea1908651326b1b81f2761218f3397_.
 
 -   ### Etherscan (https://etherscan.io)
-    https://etherscan.io/address/0x0775e11309d75aA6b0967917fB0213C5673eDf81
-    https://etherscan.io/tx/0x99c654b2338dc6e9f438b82acd5eb8af2fa2d2fe69a387714f2b2fa935ee8dbe
+    https://etherscan.io/address/0x082cCe3072A26a3871D3e5D40afB425fF5038Cf6
```
Comment (krlosMata) [inline verifyMainnetDeployment/verifyMainnetProofVerifier.md:219]:
```suggestion
    https://etherscan.io/address/0xc521580cd8586cc688a7430f9dce0f6a803f2883
```

File: verifyMainnetDeployment/verifyMainnetProofVerifier.md [hunk 2]
```diff
@@ -213,18 +213,18 @@ To download bytecode of deployed smartcontract, need the address of smart contra
...
+    https://etherscan.io/tx/0xeef8a49cc2469c11043eeb4a1a90c9c184ea1908651326b1b81f2761218f3397
 
 
     Click to show more > Input Data > Select all data and copy to clipboard.
 
 -   ### Beacocha (https://beaconcha.in)
-    https://beaconcha.in/address/0x0775e11309d75aA6b0967917fB0213C5673eDf81
-    https://beaconcha.in/tx/0x99c654b2338dc6e9f438b82acd5eb8af2fa2d2fe69a387714f2b2fa935ee8dbe
+    https://beaconcha.in/address/0x082cCe3072A26a3871D3e5D40afB425fF5038Cf6
```
Comment (krlosMata) [inline verifyMainnetDeployment/verifyMainnetProofVerifier.md:226]:
```suggestion
    https://beaconcha.in/address/c521580cd8586Cc688A7430F9DcE0f6A803F2883
```

**Review by krlosMata — CHANGES_REQUESTED** [review]

**Review by krlosMata — APPROVED** [review]
:smile_cat:

**Review by eduadiez — APPROVED** [review]
LGTM

**Pull request merged**